src/main/java/dan200/computercraft/shared/pocket/peripherals/PocketSpeaker.java

```diff
@@ -74,6 +75,7 @@ else if ( entity != null )
                 speaker.setLocation( entity.getEntityWorld(), entity.posX, entity.posY, entity.posZ );
             }
             speaker.update();
+            access.setLight( speaker.madeSound() ? 0x3320fc : -1 );
```

Comment:
Well this is colors that this one is set to use. Dunno what is your problem with that.